pkg/otlp/config_parser.go

```diff
@@ -34,41 +34,83 @@ service:
       exporters: [otlp]
 `
 
+// metricsConfig is the metrics OTLP pipeline configuration.
+// TODO (AP-1254): Set service-level configuration when available.
+const metricsConfig string = `
```

Comment:
```suggestion
const defaultMetricsConfig string = `
```
if I read the code right, this is used as the default and user config is merged over it

Reply:
You are right, renamed it!